Algorithm.Python/readme.md

```diff
@@ -26,7 +26,9 @@ Before we enable python support, follow the [installation instructions](https://
 "algorithm-location": "../../../Algorithm.Python/BasicTemplateAlgorithm.py",
 ```
  3. Rebuild LEAN. This step will ensure that the `Python.Runtime.dll` you set in 2.1 will be used.
+      Note: You should do a complete/clean build to recompile the whole solution from scratch, ignoring anything it's done before.
```

Comment:
interesting, I didn't realize the readme already said to **rebuild* lean... sounds like we just didn't follow instructions very well.